### ISSUE TITLE
Problem with downloading DEF 14A

### ISSUE THREAD

**Issue by gsccl1984** [jadchaar/sec-edgar-downloader#129]
Hi, I had an issue with downloading DEF 14A with both version 5.0.0 and 5.0.1 as it always return 0:

![image](https://github.com/jadchaar/sec-edgar-downloader/assets/147093232/9f536313-4000-4f41-8997-bc87b3197bb4)

It works fine for other format, such as 10-K or 10-Q.

Thanks

**Comment by jadchaar** [comment]
Hi @gsccl1984, I looked into this today. It seems to be an issue with the use of `rstrip` when cleaning up the amends suffix here: https://github.com/jadchaar/sec-edgar-downloader/blob/b919ddd70e8f08ed00731cb52dfde9ff5e7cab7d/sec_edgar_downloader/_orchestrator.py#L80C8-L80C8.

Upon testing, it appears that the use of `rstrip` is leading to some unintended issues when a filing type ends with any type of `A`:
```
>>> test_no_amend = "DEF 14A"
>>> test_no_amend.rstrip("/A")
DEF 14
>>> test_amend = "DEF 14A/A"
>>> test_amend.rstrip("/A")
DEF 14
```

I am working on patching this and will release the fix in v5.0.2.

**Comment by jadchaar** [comment]
v5.0.2 has just been released to address this issue. Thanks for reporting.
